### ISSUE TITLE
the MAP on NUSWIDE  is just about 0.685 when the bits length is 48

### ISSUE THREAD

**Issue by wzb1005** [thuml/DPH#2]
Hi,
I have tried the code, but the MAP on NUSWIDE  is just about 0.685 when the bits length is 48,  and it  is far from the reported result 0.7218. So why?
Yours sincerely.

**Comment by caozhangjie** [comment]
Have you tried our pretrained evaluation model?

**Comment by wzb1005** [comment]
I have tried your provided caffemodel "nuswide_81_48.caffemodel", but the MAP is just  "0.69239" . 
I have provided  the "deploy.prototxt" as an attachment for you to check.
I'm looking forward to your responding.
( If the "Dropout" layer should be  deleted??)
Thank you!
Yours Sincerely.
WZB.


[deploy.txt](https://github.com/thuml/DPH/files/3488474/deploy.txt)
(Just change the file type to ".prototxt")



------------------ 原始邮件 ------------------
发件人: "Zhangjie Cao"<notifications@github.com>;
发送时间: 2019年8月9日(星期五) 凌晨1:26
收件人: "thuml/DPH"<DPH@noreply.github.com>;
抄送: "ww"<749978334@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [thuml/DPH] the MAP on NUSWIDE  is just about 0.685 when the bitslength is 48 (#2)




Have you tried our pretrained evaluation model?

**Comment by caozhangjie** [comment]
https://github.com/thuml/DPH/blob/master/models/train/nuswide_81/train_val.prototxt The train prototxt is here.

**Comment by wzb1005** [comment]
Thank you, I know this file. And what I used is just this file.
I have found that :
the "PairwiseLossLayer" and "QuantizationLoss" have some "threshold" fields (in the "train_val.prototxt" file) that don't described in the paper. I doubt if the "prototxt" file is right or not.
